pkg/scheduler/scheduler.go

```diff
@@ -167,15 +177,15 @@ func (s *Scheduler) nominate(ctx context.Context, workloads []workload.Info, sna
 		ns := corev1.Namespace{}
 		e := entry{Info: w}
 		if cq == nil {
-			e.noAdmissionReason = "ClusterQueue not found"
+			e.inadmissibleReason = "ClusterQueue not found"
 		} else if err := s.client.Get(ctx, types.NamespacedName{Name: w.Obj.Namespace}, &ns); err != nil {
-			e.noAdmissionReason = fmt.Sprintf("Could not obtain workload namespace: %v", err)
+			e.inadmissibleReason = fmt.Sprintf("Could not obtain workload namespace: %v", err)
 		} else if !cq.NamespaceSelector.Matches(labels.Set(ns.Labels)) {
-			e.noAdmissionReason = "Workload namespace doesn't match ClusterQueue selector"
+			e.inadmissibleReason = "Workload namespace doesn't match ClusterQueue selector"
 		} else if !e.assignFlavors(log, snap.ResourceFlavors, cq) {
-			e.noAdmissionReason = "Workload didn't fit in the remaining quota"
+			e.inadmissibleReason = "Workload didn't fit in the remaining quota"
 		} else {
-			e.admissible = true
+			e.status = nominated
 			log.V(3).Info("Nominated workload for admission")
```

Comment:
Should we remove this line here and log every time for every entry at the end? Including the status and reason.

Reply:
moved the log to the loop at the end of the `schedule` func